cuegui/cuegui/JobMonitorTree.py

```diff
@@ -226,23 +226,41 @@ def setLoadMine(self, value):
         @type  value: boolean or QtCore.Qt.Checked or QtCore.Qt.Unchecked"""
         self.__loadMine = (value is True or value == QtCore.Qt.Checked)
 
-    def addJob(self, job):
+    def addJob(self, job, timestamp=None):
         """Adds a job to the list. With locking"
         @param job: Job can be None, a job object, or a job name.
         @type  job: job, string, None"""
```

Comment:
Could you add a line to the docstring explaining what the new `timestamp` param does?

Reply:
Just added timestamp to the docstring